src/Microsoft.Health.Fhir.SqlServer/Features/Schema/Migrations/57.diff.sql

```diff
@@ -0,0 +1,368 @@
+--DROP PROCEDURE dbo.MergeResourcesBeginTransaction
+GO
+CREATE OR ALTER PROCEDURE dbo.MergeResourcesBeginTransaction @Count int, @TransactionId bigint = 0 OUT, @SurrogateIdRangeFirstValue bigint = 0 OUT, @SequenceRangeFirstValue int = 0 OUT -- TODO: Remove @SurrogateIdRangeFirstValue
```

Comment:
I've noticed that you are still supporting old parameters, that's good because it makes the deployment easier to be rolled back if needed. Thanks.

Reply:
Yep, ignore my comment. SQL != C#, order of parameters isn't important.